src/ai/backend/manager/models/gql_models/session.py

```diff
@@ -191,6 +193,18 @@ class Meta:
     vfolder_mounts = graphene.List(lambda: graphene.String)
     occupied_slots = graphene.JSONString()
     requested_slots = graphene.JSONString()
+    image_references = graphene.List(
+        lambda: graphene.String,
+        description="Added in 24.12.0.",
+    )
+    image_nodes = PaginatedConnectionField(
```

Comment:
A session can have only one image, right? Please change to `image_node`

Reply:
No, a session can have multiple images. There is no use case of multi-image sessions yet